### PR TITLE
chore: add `getting-started` guide to the documentation

### LINKED ISSUE #117: docs: add a user guide on using a bot for the TF provider + needing to add workspace access in the UI for imports

### PR DESCRIPTION
**Pull request by parkedwards**
resolves https://github.com/PrefectHQ/terraform-provider-prefect/issues/117

### CODE REVIEW

File: docs/guides/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+```shell
+export PREFECT_CLOUD_API_KEY="your API Key"
+export PREFECT_CLOUD_ACCOUNT_ID="your Account/Organization ID"
+```
+
+```terraform
+provider "prefect" {}
+```
+
+The optional `account_id` and `workspace_id` attributes set default values, so that any subsequent resources will inherit those values. Set a `workspace_id` if your use case calls for managing only a single Workspace
```
Comment (jamiezieziula) [inline docs/guides/getting-started.md:43]: is `account_id` optional?

File: docs/guides/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+### Service Account Keys
+
+Most production use-cases will call for using a dedicated Service Account's API Key when invoking Terraform, as the API Key can be managed as a team - independent of any one User.
+
+<img src="../images/service-account-api-key-example.png" alt="Service Account API Key Example" align="center" width="400">
+
+<br>
+
+After creating the Service Account, the API Key will be generated and displayed for you in the UI.
+m
```
Comment (jamiezieziula) [inline docs/guides/getting-started.md:78]:
```suggestion
```

**Review by jamiezieziula — APPROVED** [review]
minor comments, lgtm!

File: docs/guides/getting-started.md (hunk 2 above)
Comment (parkedwards) [reply docs/guides/getting-started.md]: how did you even catch that

File: docs/guides/getting-started.md (hunk 1 above)
Comment (parkedwards) [reply docs/guides/getting-started.md:43]: for now, it is. all the account-scoped resources have an account_id override in them as well

**Pull request merged**